Refactor imports to be consistent with `edition = "2023_10"` (#473)
cairo-book/cairo-book: src/ch03-02-dictionaries.md

{{#include ../listings/ch03-common-collections/no_listing_07_intro/src/lib.cairo}}
```

> `Felt252DictTrait` trait doesn't need to be explicitly imported as it is included in the prelude. Therefore, all the methods we need to interact with the dictionary are by default in the scope. We can create a new instance of `Felt252Dict<u64>` by using the `default` method of the `Default` trait and add two individuals, each one with their own balance, using the `insert` method. Finally, we check the balance of our users with the `get` method.

Reviewer: ```suggestion
We can create a new instance of `Felt252Dict<u64>` by using the `default` method of the `Default` trait and add two individuals, each one with their own balance, using the `insert` method. Finally, we check the balance of our users with the `get` method. These methods are defined in the `Felt252DictTrait` trait in the core library.
```

### Dictionaries of types not supported natively

One restriction of `Felt252Dict<T>` that we haven't talked about is the trait `Felt252DictValue<T>`.

> This trait, included in the prelude, defines the `zero_default` method which is the one that gets called when a value does not exist in the dictionary.

Reviewer: ```suggestion
This trait defines the `zero_default` method which is the one that gets called when a value does not exist in the dictionary.
```
